core/common/src/jvmMain/kotlin/team/aliens/dms/kmp/core/common/timer/CountDownTimer.jvm.kt

```
@@ -0,0 +1,14 @@
+package team.aliens.dms.kmp.core.common.timer
+
+@Suppress("EXPECT_ACTUAL_CLASSIFIERS_ARE_IN_BETA_WARNING")
+actual class CountDownTimer {
+    actual fun start(
+        durationMillis: Long,
+        intervalMillis: Long,
+        listener: CountDownTimerListener,
+    ) {
+    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)